resources/logging/values.yaml

```diff
@@ -1,84 +1,151 @@
-test:
-  image:
-    name: test-logging
-
-logspout:
-  resources:
-    requests:
-      cpu: 125m
-      memory: 96Mi
-    limits:
-      memory: 400Mi
-  image:
-    repository: gliderlabs/logspout
-    tag: "v3.2.5"
-    pullPolicy: IfNotPresent
-  tolerations:
-  - effect: NoSchedule
-    operator: Exists
+rbac:
+  create: true
+  pspEnabled: true
 
-oklog:
-  image:
-    repository: oklog/oklog
-    tag: "v0.3.2"
-    pullPolicy: IfNotPresent
-  # The number of log collection service replicas. Should be a minimum of 3.
+serviceAccount:
+  create: true
+  name: 
```

Comment:
empty name?

Reply:
pod security policies are cluster wide, but they will be activated only with a service account role via verb "use"